src/screens/assets/ManageAssetsDialog/ManageAssetsDialog.tsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import { useTheme } from "@emotion/react";
+import { observer } from "mobx-react";
+
+import Button from "@components/Button.tsx";
+import { Column, Row } from "@components/Flex.tsx";
+import SideBar from "@components/SideBar.tsx";
+import Text, { TEXT_TYPES } from "@components/Text.tsx";
+import depositAssets from "@src/assets/icons/depositAssets.svg";
+import { FuelNetwork } from "@src/blockchain";
+import BN from "@src/utils/BN.ts";
+import { useStores } from "@stores";
+
+import AssetBlock from "./AssetBlock";
+import styled from "@emotion/styled";
+
+const ManageAssetsDialog = observer(() => {
+  const { quickAssetsStore, balanceStore } = useStores();
+  const theme = useTheme();
+  // const vm = useSpotTableVMProvider();
+  const bcNetwork = FuelNetwork.getInstance();
+
+  const balanceData = Array.from(balanceStore.balances)
+    .filter(([, balance]) => balance && balance.gt(0))
```

Comment:
```suggestion
    .filter(([, balance]) => balance && balance.gt(BN.ZERO))
```

Когда с 0 и big number работаем, стараюсь использовать BN.ZERO

Reply:
поправил